src/backend/renderer/utils.rs

```diff
@@ -37,8 +37,9 @@ const MAX_DAMAGE: usize = 4;
 
 impl SurfaceState {
     pub fn update_buffer(&mut self, dh: &DisplayHandle, attrs: &mut SurfaceAttributes) {
+        let _delta = attrs.buffer_delta.take();
```

Comment:
While I don't think this PR needs to use the offset everywhere to upgrade to version 5, leaving this unused here feels also not very useful. We should make it at least accessible to downstream.

What about adding a `buffer_delta` field to `SurfaceState` + some shorts docs?

Reply:
Done in: https://github.com/Smithay/smithay/pull/602/commits/00e45a9e7fb394328add121fe6ddbe48803362b4